index.html

```diff
@@ -1089,12 +1089,23 @@ <h2>
               </li>
             </ol>
           </li>
-          <li>Present a user interface that will allow the user to interact
-          with the |handlers|. The user agent SHOULD prioritize the preference
-          of the user when presenting payment methods. It is RECOMMENDED that
-          the language of the user interface match the [=node/language=] of the
-          <a data-cite="HTML/dom.html#the-body-element-2">the body element</a>
-          element.
+          <li>
+            <p>
+              Present a user interface that will allow the user to interact
+              with the |handlers|. The user agent SHOULD prioritize the
+              preference of the user when presenting payment methods.
+            </p>
+            <aside class="note" title="Localization of the payment sheet">
+              <p>
+                The API does not provide a means for developers to specify the
+                language in which the payment sheet is presented to end
+                users. As such, user agents will generally present the payment
```

Comment:
```suggestion
                users. As such, user agents will generally use the user agent's default language to present the payment sheet.
```
* Sentence was rather hard to read. Have suggested a rewording.
* `generally` doesn't seem to have a particular meaning unlike `SHOULD` and `RECOMMENDED`. In this case, it is not clear how strictly the spec wants user agents to follow the `generally` recommendation. Perhaps we could clarify this?